lib/github-release.rb

```diff
@@ -119,7 +120,7 @@ def create_release(tag)
 
 		api.create_release(repo_name, tag, :name => name, :body => body)
 
-		puts " done!"
+		puts "done!"
```

Comment:
Won't removing the leading space from that string result in a message that reads "`Creating a release for v1.2.3...done!`" (ie no space between the pseudo-ellipsis and the "done!")?